functions/src/index.ts

```diff
@@ -45,6 +45,7 @@ app.post('/helloWorld', (req: Request, res: Response) => {
 
 import { emailLogin, emailRegister } from './user'
 import { getUserData, updateUserData } from './userData'
+import { get_verdict, judgeIsOnline, submit } from './judge'
```

Comment:
super nit: mixing camelCase and snake_case is funny

Reply:
oops lol yea i can fix that